orc8r/lib/go/service/config/service_config.go

```diff
@@ -33,7 +33,7 @@ var (
 	configDir         = "/etc/magma/configs"
 	oldConfigDir      = "/etc/magma"
 	configOverrideDir = "/var/opt/magma/configs"
-	specDir           = "/etc/magma/configs/orc8r/swagger_specs"
+	specDir           = "/etc/magma/swagger/specs"
```

Comment:
Let's move all the Swagger-related logic out of lib/.../service_config.go and into the relevant cloud swagger pkg -- since specs aren't related to the config directory anymore